scripts/report_scripts/index.Rmd

```diff
@@ -93,8 +93,12 @@ library(dplyr)
 library(viridis)
 ```
 
-
 ```{r input}
+# TODO given as input? or only the scripts dir? similar variant_report
+source(params$fun_cvrg_scr)
+source(params$fun_pool)
+source(params$fun_index)
+source(params$fun_lm)
```

Comment:
This commit is fine (it's just moving things around), but there's something unsettling about sourcing files that have been passed around as arguments.  It smells wrong to me.

Reply:
Do you think we should hardcode those? I think currently you could rig snakemake up to rerun rules whose inputs have changed, which may be useful during development. I have not used that so far though.